resolwe_bio/processes/support_processors/rnaseqc_qc.py

```diff
@@ -0,0 +1,346 @@
+
+    df = pd.read_csv(
+        annotation_path,
+        sep="\t",
+        comment="#",
+        header=None,
+        usecols=[0, 2, 3, 4, 6, 8],
+        low_memory=False,
+    )
+
+    df = pd.DataFrame(
+        {
+            "chromosome": df[0].str.lstrip("chr"),
+            "feature_type": df[2],
+            "start": df[3],
+            "end": df[4],
+            "strand": df[6],
+            "gene_id": df[8].str.extract(r'gene_id "(.*?)"', expand=False).values,
+        },
+    )
+
+    df.replace("", np.nan, inplace=True)
+
+    columns = {
+        "chromosome": "first",
+        "feature_type": "first",
+        "start": "min",
+        "end": "max",
+        "strand": "first",
+        "gene_id": "first",
+    }
+
+    df["gene_id"] = (
+        "chr"
+        + df["chromosome"].astype(str)
+        + "_"
+        + df["strand"].astype(str)
+        + "_"
+        + df["gene_id"].astype(str)
+    )
+
+    genes_df = df.groupby(["gene_id"]).agg(columns).assign(feature_type="gene")
+    transcripts_df = (
+        df.groupby(["gene_id"]).agg(columns).assign(feature_type="transcript")
+    )
+    df = pd.concat([genes_df, transcripts_df, df], ignore_index=True)
+
+    df["attributes"] = df.apply(lambda x: add_attributes(x), axis=1)
+
+    out_df = pd.DataFrame(
+        {
+            "chromosome": df["chromosome"],
+            "annotation_source": ["UCSC" for _ in range(len(df))],
+            "feature_type": df["feature_type"],
+            "start": df["start"],
+            "end": df["end"],
+            "score": ["." for _ in range(len(df))],
+            "strand": df["strand"],
+            "genomic_phase": ["." for _ in range(len(df))],
+            "attributes": df["attributes"],
+        },
+    )
+
+    out_df.sort_values(by=["chromosome", "start"], ascending=[True, True], inplace=True)
+    out_df.to_csv(
+        "formatted_annotation.gtf",
+        sep="\t",
+        header=False,
+        index=False,
+        quoting=csv.QUOTE_NONE,
+    )
+
+
+class QcRnaseqc(Process):
+    """RNA-SeQC QC analysis.
+
+    More information can be found in the
+    [GitHub repository](https://github.com/getzlab/rnaseqc)
+    and in the [original paper](https://academic.oup.com/bioinformatics/article/37/18/3048/6156810?login=false)
+    The current version of RNA-SeQC is 2.4.2.
+    """
+
+    slug = "rnaseqc-qc"
+    name = "RNA-SeQC"
+    process_type = "data:rnaseqc:qc"
+    version = "1.0.0"
+    category = "QC"
+    data_name = "{{ alignment|name|default('?') }}"
+    scheduling_class = SchedulingClass.BATCH
+    persistence = Persistence.CACHED
+
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {
+            "docker": {"image": "public.ecr.aws/genialis/resolwebio/qc:1.1.0"}
+        },
+        "resources": {"cores": 1, "memory": 32768, "storage": 100},
+    }
+
+    entity = {
+        "type": "sample",
+        "input": "alignment",
+    }
+
+    description = "Quality of RNA-seq Tool-Set."
```

Comment:
Description is usually located in the docstring. Perhaps this line can be safely omitted?